### DISCUSSION TITLE
Project management

### DISCUSSION THREAD

**Discussion by lan496** [spglib/spglib#298] [General]
@atztogo @LecrisUT 
I will use the GitHub Projects to track my WIP and others for this project. If you are interested in, please try using it! 
https://github.com/orgs/spglib/projects/1/views/1

**Comment by lan496** [comment]
Currently, the columns are as follows:
- "Issue and PR": Issues and PRs are automatically added in this column
- "Being fixed in PR": Issues to be fixed in a PR
- "WIP": Work in progress
- "Require review or comment": Issues and PRs waiting a review or comment
- "Waiting": Waiting for other contributors or users
- "Done": Closed issues and PRs are automatically moved to this column

**Reply by LecrisUT** [reply]
Cool seems useful. We should also use `assign` and make that a filter for the WIP so it's easier to keep track than manually moving them around. That way e.g. if I finish the testing framework, but I need help with test samples I'll just change the `assign` for it and then back when that's done.

**Reply by lan496** [reply]
I think both `assign` and manual WIP-column will be useful because we can pass a PR card to another's WIP column for requesting code review. Anyway, I agree that `assign` should be added for ease to remember.

**Reply by LecrisUT** [reply]
Does https://github.com/orgs/spglib/projects/1/views/4 work for you? If so can we merge the WIP columns?

**Reply by lan496** [reply]
Thanks, it works! I've merged the WIP columns into one.
